src/main/java/org/trigon/hardware/misc/leds/AddressableLEDStrip.java

```diff
@@ -0,0 +1,182 @@
+package org.trigon.hardware.misc.leds;
+
+import com.ctre.phoenix.led.LarsonAnimation;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+
+import java.util.function.Supplier;
+
+public class AddressableLEDStrip extends LEDStrip {
+    private static AddressableLED LED;
+    private static AddressableLEDBuffer LED_BUFFER;
+    private int lastBreatheLED;
+    private double lastLEDMovementTime = 0;
+    private double rainbowFirstPixelHue = 0;
+    private boolean areLEDsOnForBlinking = false;
+    private boolean alternateColor = true;
+    private int amountOfColorFlowLEDs = 0;
+
+    public static void setLED(AddressableLED led) {
+        LED = led;
+    }
+
+    public static void setLEDBuffer(AddressableLEDBuffer ledBuffer) {
+        LED_BUFFER = ledBuffer;
+    }
```

Comment:
Why are these public?

Reply:
So you can use them. Why shouldn't they be public